sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala

```diff
@@ -328,11 +320,12 @@ private[ui] class SqlStatsPagedTable(
       <td>
         {if (info.closeTimestamp > 0) formatDate(info.closeTimestamp)}
       </td>
+      <!-- Returns a human-readable string representing a duration such as "5 second 35 ms"-->
       <td >
-        {UIUtils.formatDuration(executionTime)}
+        {formatDurationVerbose(executionTime)}
```

Comment:
Does this change the format at all? it's fine to change this if it's making it more consistent.

Reply:
Yes. For example  5 sec 356 ms, `formatDuration` will  give 5.3 s, whereas `formatDurationVerbose` will give 5 sec 356 ms. I think, we should change this in other pages too.

Reply:
I'm not sure. If it's consistently in the shorter more human readable format already, I'd leave it.

Reply:
Yes, but in thriftserver page, earlier it was `UIUtils.formatDurationVerbose`. For consistency, I think I added here.

Reply:
You're saying that it was `formatDurationVerbose` before the commit you made yesterday? if so, OK, I can see restoring it. I'm mostly concerned with being consistent too.

Reply:
Yes, Right.